frontend/src/app/modules/bim/ifc_models/bcf/new-split/bcf-new-split.component.ts

```diff
@@ -31,6 +31,7 @@ import {Component} from '@angular/core';
 import {WorkPackageResource} from "core-app/modules/hal/resources/work-package-resource";
 import {InjectField} from "core-app/helpers/angular/inject-field.decorator";
 import {IFCViewerService} from "core-app/modules/bim/ifc_models/ifc-viewer/ifc-viewer.service";
+import {HalResource} from "core-app/modules/hal/resources/hal-resource";
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/993bb07a53e559722c753352294812e39adfbcd2/frontend/src/app/modules/bim/ifc_models/bcf/new-split/bcf-new-split.component.ts#L35"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>